app/src/main/java/com/example/kotlin_java_practicalss/AbstractClass_demo.java

```diff
@@ -0,0 +1,33 @@
+package com.example.kotlin_java_practicalss;
+
+abstract class jaguar
+{
+    jaguar()
```

Comment:
Starting brace should be written at same line of function declaration,
jaguar() {
    // function body ( here for this specific scenario:constructor body )
}